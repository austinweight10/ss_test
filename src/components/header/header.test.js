import React from "react"
import Enzyme, { shallow } from "enzyme"

import Adapter from "enzyme-adapter-react-16"

import Header from "./"

Enzyme.configure({ adapter: new Adapter() })

describe("Header", () => {
  const wrapper = shallow(<Header />)

  it("Header has logo", () => {
    expect(wrapper.find("[data-test='logo_img']").prop("src")).toEqual(
      "/SoleLogo.png"
    )
  })

  it("Header has link", () => {
    expect(wrapper.find("[data-test='logo_link']").prop("to")).toEqual("/")
  })

  it("Header matches shapshot", () => {
    expect(wrapper).toMatchSnapshot()
  })
})
