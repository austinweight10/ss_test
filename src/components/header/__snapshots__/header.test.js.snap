// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header matches shapshot 1`] = `ShallowWrapper {}`;
